src/omv/boards/ARDUINO_GIGA/imlib_config.h

```diff
@@ -131,6 +131,7 @@
 // Enable Tensor Flow
 #if !defined(CUBEAI)
 #define IMLIB_ENABLE_TF (IMLIB_TF_DEFAULT)
+#define IMLIB_ENABLE_TF_YOLO
```

Comment:
It seems like the default ops level supports YOLO, correct ? If so this option doesn't matter we should just enable yolo detection by default. Disabling it only removes very little code in `py_tf.c`.

Reply:
You're not going to be running YOLO on any board without SDRAM. So, we'd just be using Flash for no value on the Nicla/H7/M7.

Reply:
Good point.